spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -13,5 +13,49 @@
         expect(response.status).to eq(200)
       end
     end
+
+    describe "GET /admin/users/edit/1" do
+      it 'works!' do
+        target = create(:default_user)
+        get 'edit', params: { id: target.id }
+        expect(response.status).to eq(200)
+      end
+    end
+
+    describe 'POST /admin/users/update' do
+      it 'works!' do
+        target = create(:default_user)
+        data = User.find(target.id)
+        update_name = 'SushiKuitai'
+        data[:display_name] = update_name
+        keys = %w[id username display_name email biography admin]
+        data_attr = data.attributes
+        data_attr.each do |k, v|
+          data_attr.delete(k) unless keys.include?(k)
+        end
+        post :update, params: { user: data_attr }
+        new_name = User.find(target.id).display_name
+        expect(new_name).to eq(update_name)
+        expect(response.status).to eq(302)
+      end
+    end
+
+    describe 'POST /admin/users/update' do
+      it 'move to edit screen' do
+        target = create(:default_user)
+        data = User.find(target.id)
+        update_name = '' # validation error
+        data[:display_name] = update_name
+        keys = %w[id username display_name email biography admin]
+        data_attr = data.attributes
+        data_attr.each do |k, v|
```

Comment:
__[RuboCop]__

Lint/UnusedBlockArgument: Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.


<!-- Comment by [SideCI](https://sideci.com) -->